docs/src/inputs/prompt/examples/exampleInformativePrompt.jsx

```diff
@@ -0,0 +1,48 @@
+import {
+    Button,
+    Icon,
+    Prompt,
+} from 'react-cm-ui';
+import React, { useState } from 'react';
+
+function ExampleInformativePrompt() {
+    const [showPrompt, setShowPrompt] = useState(false);
+
+    function onPromptClick() {
+        setShowPrompt(true);
+    }
+
+    function onYesClick() {
+        setShowPrompt(false);
+    }
+
+    function onNoClick() {
+        setShowPrompt(false);
+    }
+
+    return (
+        <div>
+            <Prompt
```

Comment:
This example needs all three states show as in: 

![image](https://user-images.githubusercontent.com/1100790/98309038-578b0100-1f7e-11eb-9ec7-c9ccbbf2249f.png)

Reply:
Fixed